### DISCUSSION TITLE
Adding Leaflet-Geoman drawing

### DISCUSSION THREAD

**Discussion by Falke-Design** [tomickigrzegorz/leaflet-examples#64] [Announcements]
https://github.com/geoman-io/leaflet-geoman

Demo: https://jsfiddle.net/o1dwu2vg/

**Comment by tomickigrzegorz** [comment]
We'll see 😂

[leaflet-geoman](https://tomik23.github.io/leaflet-examples/#66.leaflet-geoman)
